coding/internal/file_data.cpp

```diff
@@ -44,7 +44,7 @@ FileData::FileData(string const & fileName, Op op)
   if (op == OP_WRITE_EXISTING)
   {
     // Special case, since "r+b" fails if file doesn't exist.
-    m_File = fopen(fileName.c_str(), "wb");
+    m_File = fopen(fileName.c_str(), "w+b");
```

Comment:
w+b truncates file to 0 length or creates for reading and writing. wb truncates to 0 length or creates for writing. What the point in your change?